scripts/add-chart-version

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+usage() {
+  echo "Usage: CHARTS=<chart1,chart2> TEMPLATE_BASE_URL=https://raw.githubusercontent.com/rancher/your_chart TEMPLATE_VALUES='KEY1=value1' $0"
+}
+
+if [ "x${CHARTS}" = "x" ]; then
+  usage
+  exit 1
+fi
+
+if [ "x${TEMPLATE_BASE_URL}" = "x" ]; then
+  usage
+  exit 1
+fi
+
+if [ "x${TEMPLATE_VALUES}" = "x" ]; then
+  usage
+  exit 1
+fi
```

Comment:
Just an approach to avoid empty strings, https://tldp.org/LDP/abs/html/comparison-ops.html; 

```
As S.C. points out, in a compound test, even quoting the string variable might not suffice. [ -n "$string" -o "$a" = "$b" ] may cause an error with some versions of Bash if $string is empty. The safe way is to append an extra character to possibly empty variables, [ "x$string" != x -o "x$a" = "x$b" ] (the "x's" cancel out).
```